pages/item-new/_.vue

```diff
@@ -35,31 +35,23 @@
       -->
         <client-only />
       </b-container>
-      <b-container>
+      <b-container class="mw-1024">
         <b-row class="mb-3">
           <b-col>
-            <!-- TODO: add new title + description box
-              Useful data:
-
-              titlesInCurrentLanguage
-              descriptionInCurrentLanguage
-            -->
+            <SummaryInfo
+              :description="descriptionInCurrentLanguage"
+              :titles="titlesInCurrentLanguage"
+            />
           </b-col>
         </b-row>
-        <b-row class="mb-3">
+        <b-row
+          v-if="relatedEntities && relatedEntities.length > 0"
+        >
```

Comment:
When there are related entities, the whitespace looks okay. But when there aren't any, for example http://localhost:3000/en/item-new/327/_00000070p, it could perhaps use some extra whitespace between title/description-box and metadata. What do you think?

Reply:
ok, I didn't think it was too bad but I've add in a buffer row if there's no related
```
<b-row
    v-else
    class="mb-3"  
></b-row>
```